src/WarcraftLegacies.Source/Quests/Dalaran/QuestNewGuardian.cs

```diff
@@ -18,7 +18,7 @@ public sealed class QuestNewGuardian : QuestData
       @"ReplaceableTextures\CommandButtons\BTNAstral Blessing.blp")
     {
       _jaina = jaina;
-      AddObjective(new ObjectiveControlCapital(dalaran, false));
+      AddObjective(new ObjectiveCapitalAlive(dalaran));
```

Comment:
This works way better but, looking at the list of Objectives, I actually don't think we need a new Objective for it. Use `ObjectiveUnitAlive` instead - it's the same one used in Dalaran's Theramore quest anyway.

Reply:
Gotcha. Shall  I remove the ObjectiveCapitalAlive file now also as its now not it in use

Reply:
Yup. The reason we have a distinct `ObjectiveLegendAlive` objective rather than just using `ObjectiveUnitAlive` is because Legends can sometimes change units, whereas Capitals always have the same unit. So `ObjectiveCapitalAlive` isn't needed.